src/cli-validator/utils/processConfiguration.js

```diff
@@ -367,9 +367,39 @@ const validateConfigOption = function(userOption, defaultOption) {
   return result;
 };
 
+const getSpectralRuleset = async function(defaultRuleset) {
+  // List of ruleset files to search for
+  const ruleSetFilesToFind = [
+    '.spectral.yaml',
+    '.spectral.yml',
+    '.spectral.json'
+  ];
+  let ruleSetFile;
+
+  // search up the file system for the first ruleset file found
+  try {
+    for (const file of ruleSetFilesToFind) {
+      if (!ruleSetFile) {
+        ruleSetFile = await findUp(file);
+      }
+    }
+  } catch (err) {
```

Comment:
This is an extremely minor comment but since you aren't using the error at all, I think it might be better style to just use `catch ()`

Reply:
I had to change this to ``catch { }``